### ISSUE TITLE
crash on completing ConfigParser's args with bpython-urwid on python 3

### ISSUE THREAD

**Issue by glyph** [bpython/bpython#721]
- `pip install bpython`
- `pip freeze`
  ```
  blessings==1.6
  bpython==0.16
  certifi==2017.11.5
  chardet==3.0.4
  curtsies==0.2.11
  greenlet==0.4.12
  idna==2.6
  Pygments==2.2.0
  requests==2.18.4
  six==1.11.0
  urllib3==1.22
  urwid==1.3.1
  wcwidth==0.1.7
  ```
- `bpython-urwid`
- ```
  >>> from configparser import ConfigParser
  >>> ConfigParser(
  ```

```
Traceback (most recent call last):
  File "/Users/glyph/.virtualenvs/tmp-20edfb013e08472/bin/bpython-urwid", line 11, in <module>
    sys.exit(main())
  File ".../site-packages/bpython/urwid.py", line 1317, in main
    myrepl.main_loop.screen.run_wrapper(run_with_screen_before_mainloop)
  File ".../site-packages/urwid/display_common.py", line 763, in run_wrapper
    return fn()
  File ".../site-packages/bpython/urwid.py", line 1268, in run_with_screen_before_mainloop
    myrepl.main_loop.run()
  File ".../site-packages/urwid/main_loop.py", line 278, in run
    self._run()
  File ".../site-packages/urwid/main_loop.py", line 376, in _run
    self.event_loop.run()
  File ".../site-packages/urwid/main_loop.py", line 682, in run
    self._loop()
  File ".../site-packages/urwid/main_loop.py", line 715, in _loop
    alarm_callback()
  File ".../site-packages/urwid/main_loop.py", line 164, in cb
    callback(self, user_data)
  File ".../site-packages/bpython/urwid.py", line 1016, in <lambda>
    0, lambda *args: self._populate_completion())
  File ".../site-packages/bpython/urwid.py", line 821, in _populate_completion
    widget_list.append(urwid.Text(markup))
  File ".../site-packages/urwid/widget.py", line 836, in __init__
    self.set_text(markup)
  File ".../site-packages/urwid/widget.py", line 879, in set_text
    self._text, self._attrib = decompose_tagmarkup(markup)
  File ".../site-packages/urwid/util.py", line 375, in decompose_tagmarkup
    tl, al = _tagmarkup_recurse(tm, None)
  File ".../site-packages/urwid/util.py", line 395, in _tagmarkup_recurse
    tl, al = _tagmarkup_recurse( element, attr )
  File ".../site-packages/urwid/util.py", line 413, in _tagmarkup_recurse
    return _tagmarkup_recurse( element, attr )
  File ".../site-packages/urwid/util.py", line 416, in _tagmarkup_recurse
    raise TagMarkupException("Invalid markup element: %r" % tm)
urwid.util.TagMarkupException: Invalid markup element: None
```

**Comment by sebastinas** [comment]
Please try again with 0.17 or current git HEAD. I cannot reproduce this issue.

**Comment by thomasballinger** [comment]
Looks like the same thing that fixed https://github.com/bpython/bpython/issues/672 fixes this; both kw-only parsing issues, in 8ac18df3a82 and now that 0.17 is released this is fixed.

Thanks for reporting both of these @glyph, sorry the release took so long.
